web/dev/sass/views/_modalRegisterLogin.sass

```diff
@@ -0,0 +1,66 @@
+@import "imports"
+
+.loginRegisterModal
+  height: 50em
+  font-family: $nav_bar-font
+  text-align: center
+  h1
+    font-weight: bold
+    color: $font-color3
+    font-size: 3rem
+  #loginFormIntro, #registerFormIntro
+    margin-top: 3rem
+  #loginFormBlock
+    height: 63rem
+    border-right: 1px solid $bg-color1
+
+
+#fosLoginForm
+  margin-top: 7rem
+  label
+    margin-bottom: 4rem
+    width: 14rem
+    text-align: right
+    margin-right: 1rem
+    margin-left: -10rem
```

Comment:
On your label you have 3 parameters for the margin option. Make just one like "margin: 0 1rem 4rem -10rem"